### PR TITLE
Add survey index and style code

### PR DESCRIPTION
**Pull request by Enekwestanley**
This is a project to design a survey form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div class="main">
```
Comment (Enekwestanley) [inline index.html:10]: Use main or section tag instead of a div to wrap the container

**Review by Enekwestanley — COMMENTED** [review]
You are almost there, apply the changes made below.

File: index.html (hunk 1 above)
Comment (Danieldotcomcoder) [reply index.html:10]: Kindly try to use more friendly words like 'please' or 'kindly' to make your comment nice and friendly 😄 👍

Comment (Danieldotcomcoder) [reply index.html:10]: Kindly explain why you suggested this change, it might be obvious to you but not to the student 👍 😄

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @Enekwestanley, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Used Inline comment ✔✔

## Required Changes ♻️
_Check the comments under the review._

* Please add at least one more inline comment as stated by the project requirments.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (Enekwestanley) [inline index.html:10]: Good job so far,
Please, try to use main or section tag to wrap the container. This makes your code semantics.

**Review by Enekwestanley — COMMENTED** [review]
Please, apply the changes and you are good to go.

**Review by smunozmo — APPROVED** [review]
Hi @Enekwestanley , 👋 

Good job adding some explanation to your review and using more friendly words, this makes your reviews easier to read and understand.
Your project is complete! Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**